buildenv/jenkins/common/build.groovy

```diff
@@ -300,6 +300,19 @@ def archive_sdk() {
         def testDir = "test"
 
         dir(OPENJDK_CLONE_DIR) {
+            // Code coverage files archive for report
+            if (params.CODE_COVERAGE) {
+                // Use WORKSPACE since code coverage gcno files are in vm folder, java is in jdk and images folder, and omr and openj9 folders are in workspace.
+                def codeCoverageDir = "${env.WORKSPACE}"
+                if (SPEC.contains('zos')) {
+                    sh "touch ${CODE_COVERAGE_FILENAME}"
+                    sh "pax --exclude=${CODE_COVERAGE_FILENAME} -wvzf  ${CODE_COVERAGE_FILENAME} ${codeCoverageDir}"
```

Comment:
Why the extra space after `-wvzf`?

Reply:
The extra space has been removed. Thanks.